task_02/index.html

```diff
@@ -1,11 +1,195 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
-</head>
-<body>
-    Task 2
-</body>
-</html>
\ No newline at end of file
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="wrapper">
+      <!-- Desktop navigation -->
+      <nav role="navigation" class="desktop-navigation">
+        <div class="nav-wrapper">
+          <div class="list-item"><a href="" class="links">Home</a></div>
+          <div class="list-item"><a href="" class="links">Approach</a></div>
+          <div class="list-item"><a href="" class="links">Work</a></div>
+          <div class="list-item"><a href="" class="links">People</a></div>
+          <div class="list-item">
+            <a href="" class="links" id="disabled">Jobs</a>
+          </div>
+          <div class="list-item">
+            <a href="" class="links" id="disabled">Blog</a>
+          </div>
+          <div class="list-item">
+            <a href="" class="links" id="disabled">Contact</a>
+          </div>
+        </div>
+      </nav>
+
+      <!-- Mobile nav -->
+      <nav role="navigation" class="mobile-navigation">
+        <div id="menuToggle">
+          <!--
+        A fake / hidden checkbox is used as click reciever,
+        so you can use the :checked selector on it.
+        -->
+          <input type="checkbox" id="toggleCheckbox" onclick="moveText()" />
```

Comment:
Same for closing tags:

https://www.w3schools.com/tags/tag_input.asp

> In HTML, the <input> tag has no end tag.

> In XHTML, the <input> tag must be properly closed, like this <input />.

Reply:
Will be fixed.